migrations.go

```diff
@@ -301,4 +303,104 @@ ALTER TABLE apps ADD COLUMN exposure TEXT NOT NULL default 'private'`,
 			`ALTER TABLE processes ADD COLUMN command text not null`,
 		}),
 	},
+	{
+		ID: 15,
+		Up: func(tx *sql.Tx) error {
+			_, err := tx.Exec(`ALTER TABLE slugs ADD COLUMN formation jsonb`)
+			if err != nil {
```

Comment:
Might be nice to have a comment about what each migration is about going forward, just to give some context.